src/components/ServiceList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import FilterList from "../components/FilterList";
+import useServices from "../hooks/useServices";
+import ServiceCard from "../components/ServiceCard";
+
+const ServiceList = () => {
+  const apiResponse = useServices();
+  const serviceItems = apiResponse[0];
```

Comment:
see comment in useServices, for this to be simplified.

Reply:
done